src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -471,38 +497,66 @@ export const TxReceiptUI = ({
           recipient={rawTransaction.to}
         />
       </div>
+      {txType === ITxType.FAUCET && (
```

Comment:
Can we add a prop for a custom button instead of having more conditionals in the component?